apollo-router/src/plugins/connectors/http_json_transport.rs

```diff
@@ -143,48 +140,24 @@ fn make_uri(
         .extend(
             template
                 .interpolate_path(inputs)
-                .map_err(HttpJsonTransportError::TemplateGenerationError)?,
+                .map_err(|err| HttpJsonTransportError::TemplateGenerationError(err.message))?,
         );
 
     let query_params = template
         .interpolate_query(inputs)
-        .map_err(HttpJsonTransportError::TemplateGenerationError)?;
+        .map_err(|err| HttpJsonTransportError::TemplateGenerationError(err.message))?;
     if !query_params.is_empty() {
         url.query_pairs_mut().extend_pairs(query_params);
     }
     Ok(url)
 }
 
-/// Flatten map-of-maps to a map of dot-delimited keys and built-in JSON values.
-fn flatten_keys(inputs: &IndexMap<String, Value>) -> Map<ByteString, Value> {
```

Comment:
Looks like we had a merge conflict here - I added a new test for `flatten_keys` below, and you removed the function here, so this no longer compiles.